components/js/SlickGrid.ts

```diff
@@ -143,7 +143,7 @@ export class SlickGrid implements OnChanges, OnInit, OnDestroy, AfterViewInit {
 
     @Output() loadFinished: EventEmitter<void> = new EventEmitter<void>();
     @Output() editingFinished: EventEmitter<any> = new EventEmitter();
-    @Output() contextMenu: EventEmitter<{x: number, y: number}> = new EventEmitter<{x: number, y: number}>();
+    @Output() contextMenu: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
If possible, it would be best to keep this typed rather than `any` so anyone using this repo (read: us) have proper intellisense. However, [the API](https://github.com/mleibman/SlickGrid/wiki/Grid-Events) doesn't even specify what fields `event` has... We could create our own interface, if needed.